src/main/kotlin/org/wycliffeassociates/resourcecontainer/ResourceContainer.kt

```diff
@@ -22,12 +23,16 @@ interface Config {
     fun write(writer: OutputStream)
 }
 
+/**
+ *  This is an object that holds resource until it is closed. It is strongly advised to
+ *  use within a disposable use() block or manually invoke the close() method.
+ */
 class ResourceContainer private constructor(val file: File, var config: Config? = null) : AutoCloseable {
 
     lateinit var manifest: Manifest
     var media: MediaManifest? = null
 
-    val accessor: IResourceContainerAccessor = when (file.extension) {
+    private val accessor: IResourceContainerAccessor = when (file.extension) {
```

Comment:
This accessor is used internally within the ResourceContainer class

Reply:
let's not make this change immediately. Hopefully things aren't using it, but this is a breaking change if they are